src/dom/attributes/events/init-event-attr.js

```diff
@@ -1,3 +1,20 @@
+/**
+ * ##Call Operation when Run Created
```

Comment:
Not necessarily on "run created" - Call operation as soon as this element is added to the DOM.